### PR TITLE
Karhu Tower

### PR DESCRIPTION
**Pull request by MaeTobiGeri**
Added A Working Tower for the Karhu Anti cheat also works on other anti cheats like Vulcan with some additional configuring. It Works by Using a Timer and a negative motion/pull down in combination with each other

https://github.com/CCBlueX/LiquidBounce/assets/167036593/63bf2265-e4f9-45d7-a567-305d824a4608

### CODE REVIEW

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/world/scaffold/tower/ScaffoldTowerKarhu.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import net.ccbluex.liquidbounce.event.events.PlayerJumpEvent
+import net.ccbluex.liquidbounce.event.sequenceHandler
+import net.ccbluex.liquidbounce.features.module.modules.world.scaffold.ModuleScaffold
+import net.ccbluex.liquidbounce.features.module.modules.world.scaffold.ModuleScaffold.towerMode
+import net.ccbluex.liquidbounce.utils.client.Timer
+import net.ccbluex.liquidbounce.utils.kotlin.EventPriorityConvention
+import net.ccbluex.liquidbounce.utils.kotlin.Priority
+
+object ScaffoldTowerKarhu : Choice("Karhu") {
+    val TimerSpeed by float("Timer", 5f, 0.1f..10f)
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/world/scaffold/tower/ScaffoldTowerKarhu.kt]: please follow kotlin naming conventions, it should be `timerSpeed`

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/world/scaffold/tower/ScaffoldTowerKarhu.kt [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    private val triggerMotion by float("Trigger", 0.06f, 0.0f..0.2f, "Y/v")
+    private val Pulldown by boolean("Pulldown", true)
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/world/scaffold/tower/ScaffoldTowerKarhu.kt]: please follow kotlin naming conventions, it should be `pulldown`

**Comment by MaeTobiGeri** [comment]
All errors in regards to naming conventions should be fixed now.

**Comment by 1zun4** [comment]
Looks good.

**Comment by EclipsesDev** [comment]
btw isn't this just a pulldown tower?

**Pull request merged**